addons/attributes/stringtable.xml

```diff
@@ -238,5 +238,11 @@
             <English>Abilities</English>
             <Polish>Zdolności</Polish>
         </Key>
+        <Key ID="STR_ZEN_Attributes_OpenAttributesDisplay">
+            <English>Open Attributes Display</English>
+        </Key>
+        <Key ID="STR_ZEN_Attributes_OpenAttributesDisplay_Description">
+            <English>Opens the Zeus attribute display for the mouse-overed or selected entity.</English>
```

Comment:
"attributes" to match the display name.

Reply:
https://github.com/ampersand38/ZEN/commit/d557e41e9009ab0fdff9e854fc4fccd299171685